pkg/metrics/metrics.go

```diff
@@ -0,0 +1,183 @@
+package metrics
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"strings"
+
+	corelisters "k8s.io/client-go/listers/core/v1"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+
+	configv1 "github.com/openshift/cluster-samples-operator/pkg/apis/samples/v1"
+	"github.com/openshift/cluster-samples-operator/pkg/client"
+	sampoplisters "github.com/openshift/cluster-samples-operator/pkg/generated/listers/samples/v1"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	separator       = "_"
```

Comment:
nit - "separator" is longer to type than "_"

Reply:
LOL .... but I don't have to hit the shift key whey typing `separator` ;-)

Reply:
it also helps to ensure we are consistent about which separator character we use, especially if you're trying to construct strings for comparison and such.